pinot-query-planner/src/test/resources/queries/PhysicalOptimizerPlans.json

```diff
@@ -307,6 +307,59 @@
       }
     ]
   },
+  "physical_opt_group_trim_enabled": {
```

Comment:
FYI: The following 3 tests are taken from the existing feature (but with physical optimizer disabled): https://github.com/apache/pinot/blob/56f1ce734bce3b46636d457f04e5bb72081a3d2a/pinot-query-planner/src/test/resources/queries/GroupByPlans.json#L253-L303